include/effects_internal.h

```diff
@@ -25,12 +26,137 @@ void shim_guTranslateF(float mf[4][4], float x, float y, float z);
 void shim_guPositionF(float mf[4][4], float r, float p, float h, float s, float x, float y, float z);
 void shim_mem_clear(void* data, s32 numBytes);
 void shim_transform_point(Matrix4f mtx, f32 inX, f32 inY, f32 inZ, f32 inS, f32* outX, f32* outY, f32* outZ, f32* outS);
-
-s32 shim_load_effect(s32 effectIndex);
-
 s32 shim_npc_raycast_down_sides(s32, f32*, f32*, f32*, f32*);
 void shim_sfx_play_sound_at_position(s32 soundID, s32 value2, f32 posX, f32 posY, f32 posZ);
 
-void fx_6_main(s32 type, f32 x, f32 y, f32 z, f32 arg4);
+#define FX_MAIN(name) void name(s32); // the actual signature is going to be wrong, but it doesn't matter
```

Comment:
Can you `undef` this at the end of the file

Reply:
undef FX_MAIN? why?